js/reveal.js

```diff
@@ -342,6 +346,7 @@
 		if( typeof query['dependencies'] !== 'undefined' ) delete query['dependencies'];
 
 		// Copy options over to our config object
+		if( typeof(options) === 'object' && options.hasOwnProperty('overrideLayout') && options.overrideLayout ) config.display = '';
```

Comment:
Can we remove this? Feels a little unexpected that disabling the layout is mutually exclusive with setting the display style style.

Reply:
Yeah, it didn't specifically impact the override... I just didn't want to see it added. :)